packages/vscode-graphql/README.md

```diff
@@ -137,6 +142,24 @@ module.exports = {
 Notice that `documents` key supports glob pattern and hence `["**/*.graphql"]`
 is also valid.
 
+Normally, you want to point your `documents` to different files per project to ensure only one schema is used for the queries. However, you can also mix queries for different schemas in the same file by adding a `#graphql:<suffix>` comment to the query that matches the `languageService.gqlTagOptions.annotationSuffix` for the project:
```

Comment:
```suggestion
Normally, you would point your `documents` to multiple-files-per-project to ensure that only one schema is used for the queries. However, you can also mix queries for different schemas into the same file by adding a `#graphql:<suffix>` comment to each query, matching the `languageService.gqlTagOptions.annotationSuffix` for the project:
```

Reply:
I think `multiple` here is not correct 🤔
What I wanted to say is that each project in `.graphqlrc` should point to different sets of files -- i.e. no overlap between projects.

Reply:
with graphql config, this is kind of true, but not always. you could have both some shared schema files and seperate schema files between two projects. you could have shared fragments declared in documents config but seperate queries. i will show some examples soon